backend/DB.py

```diff
@@ -0,0 +1,126 @@
+import mysql.connector
+from datetime import datetime, timedelta
+ 
+
+
+
+
+class SQL(object):
+    def __init__(self):
+         self.delay_time =  600
+        #  timedelta(minutes=5)
+         self.mydb = mysql.connector.connect(
+            host="localhost",
+            user="roma",
+            password="Aa123456",
+            database="ping_management"
+          )
+
+    def addCompToDB(self, compName):
+          mycursor = self.mydb.cursor()
+          sql = "INSERT INTO ping_management.comps (compName)  VALUES (%s); "
+          print(sql)
+          val = (compName,)#!!!
+          print(val)
+          mycursor.execute(sql, val)
+          self.mydb.commit()
+          print(mycursor.rowcount, "record inserted.")
+
+    def addNewPings(self, data):
+        compName = data['test_url']
+        ping = data['ping']
+        timeStem = datetime.now()
+        distComps = self.getAllComps()
+        if compName not in distComps:
+          self.addCompToDB(compName)
+        self.addPingToDB(compName, ping, timeStem)
+
+    def addPingToDB(self,compName, ping, timeStem):
+        mycursor = self.mydb.cursor()
+        sql = "INSERT INTO ping_management.pings (compName, ping, timeOfResponce)  VALUES (%s, %s, %s); "
+        val = (compName, ping, timeStem)
+        mycursor.execute(sql, val)
+        self.mydb.commit()
+        print(mycursor.rowcount, "record inserted.")
+
+    def deleteComp(self, compName):        #letaken
+        mycursor = self.mydb.cursor()
+        sql = "DELETE FROM pings WHERE compName = \'" + compName + "\'"
```

Comment:
do the linking using  an ID field

Reply:
Why?!
I need to delete those specific pings